single.php

```diff
@@ -12,7 +12,7 @@
 
 		<?php while ( have_posts() ) : the_post(); ?>
 
-			<?php get_template_part( 'content', 'single' ); ?>
+			<?php get_template_part( 'content-single', get_post_format() ); ?>
```

Comment:
If we do it this way then we can delete `content-single.php` but will need to copy the differences to `content.php`.

Do you know why Twenty Fifteen and Twenty Fourteen did not include `content-single.php`?

My reasoning for keeping it is that it give a bit more flexibility.